morphic.js

```diff
@@ -5124,6 +5159,11 @@ CursorMorph.prototype.processKeyDown = function (event) {
     this.target.escalateEvent('reactToKeystroke', event);
 };
 
+CursorMorph.prototype.processKeyUp = function (event) {
+    this.world().currentKey = null;
```

Comment:
wait! Aren't we doing that already in the ```ClipboardHandler```? Maybe we don't do it there and only here? Would that work?